__tests__/news.test.js

```diff
@@ -48,3 +48,51 @@ describe("/api/topics", () => {
     });
   });
 });
+
+//Test 3 - Get all articles
+describe("/api/articles/", () => {
+  describe("GET Article object", () => {
+    test("200: responds with an Object of articles", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body;
+          expect(articles).toBeInstanceOf(Object);
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          });
+        });
+    });
+  });
+});
+
+//Test 4 - Get articles by ID
+describe("/api/articles/:article_id", () => {
+  describe("GET Article object by specific article ID", () => {
+    test("200: responds with an Object of a specific article", () => {
+      return request(app)
+        .get("/api/articles?article_id=1")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body;
+          expect(articles).toBeInstanceOf(Object);
+          expect.objectContaining({
+            author: expect.any(String),
```

Comment:
if we are requesting an article with a specific id we don't want to be expecting any value, we are expecting the specific information relevant to that article so need to adjust these tests to match that